duet/views/kpi_dau.view.lkml

```diff
@@ -128,13 +128,36 @@ view: kpi_dau {
     filters: [period_filtered_measures: "last"]
   }
 
+  measure: dau_goal {
+    view_label: "KPI filtered metrics"
```

Comment:
Could we make this name more explicit? E.g. KPI_metrics_with_target or KPI_target_metrics?